pkg/runtime/config.go

```diff
@@ -83,6 +85,7 @@ func NewRuntimeConfig(
 		APIListenAddresses:  apiListenAddresses,
 		ApplicationProtocol: Protocol(appProtocol),
 		Mode:                modes.DaprMode(mode),
+		EnablePlacement:     enablePlacement,
 		PlacementAddresses:  placementAddresses,
```

Comment:
Can we use placement address value instead of adding a new config? If placementAddresses is empty, then it means it is disabled.

Reply:
I think it is still necessary to let the user perceive whether the placement is actively closed. If the placement is opened but the placement address is empty, it will be an error.
What do you think？
@artursouza